### ISSUE TITLE
Allow users to add information about connections on their own devices

### ISSUE THREAD

**Issue by UBIpromoter** [BrightID/BrightID#259]
Let users add notes that are just stored locally by the user.
So users can add identifying information about the person they connected to in case the supplied name and picture is not sufficient for them to be confident about remembering the person.

**Comment by adamstallard** [comment]
This is different from #249 right ?  Is the difference that this one involves a user adding notes about their connections and #249 is about users adding info about themselves that gets shared whenever they make a connection?

**Comment by UBIpromoter** [comment]
Right, #249 is about adding information about yourself that you want to share when you make new connections. For example, I could choose to add my twitter handle so everyone could follow me and to help prove who I am. This info will end up on the devices of everyone you connect to.

This #259 is about a user adding personal notes about the connections they make. So if I meet a new person I could write things like where we met, what they do, a piece of contact info, or an anecdote to help me remember who they are. This information would only be on the local device of the person who wrote the notes. The app would not facilitate the sharing of this information to anyone else.
